pkg/build/builder/source_test.go

```diff
@@ -162,6 +162,7 @@ func (r *testGitRepo) addCommit() error {
 }
 
 func TestUnqualifiedClone(t *testing.T) {
+	t.Skip("This test as its failing permanently due to Git upgrade after a CVE.Similar tests happen on other places hence skipping it")
```

Comment:
Be specific, and reference JIRA or other identifiers where appropriate. Ditto for the other comments.

```suggestion
	t.Skip("OCPBUGS-45321: git file:// protocol no longer allowed by default. See CVE-2022-39253")
```